ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -1,5 +1,75 @@
 [
   {
+    "id": "java-mysql",
+    "creator": "ide",
+    "name": "Java-MySQL",
+    "description": "Multi-machine environment with Default Java Stack and MySQL database",
+    "scope": "general",
+    "tags": [
+      "Java 1.8, Tomcat 8, MySQL 5.7"
+    ],
+    "components": [
```

Comment:
@TylerJewell Tag appear in UI and they are used to match stacks and filters. Components aren't used at all now in master.

This is a broader question and does not affect this PR only. This is how things work now, you may check with @ashumilova if you have doubts.